src/rules/require-baseline.js

```diff
@@ -549,6 +552,45 @@ export default {
 				supportsRules.pop();
 			},
 
+			"Atrule[name=media] > AtrulePrelude > MediaQueryList > MediaQuery > Condition"(
+				node,
+			) {
+				for (const child of node.children) {
+					if (child.type !== "Feature") {
+						continue;
+					}
+
+					const conditionLevel = mediaConditions.get(child.name);
```

Comment:
`conditionLevel` can be `undefined` if someone is using an unknown media condition. In that case, we should just exit early because `no-invalid-at-rules` should catch it.

Reply:
Good point! I've fixed it.
[Add check for invalid at-rules](https://github.com/eslint/css/pull/49/commits/2b8a1ab60ba17a84cca152df5af786bffe2dc0bc)